platforms.h

```diff
@@ -54,6 +68,7 @@ struct pv_platform {
 	struct pv_state *state;
 	bool mgmt;
 	bool updated;
+	struct dl_list drivers;
```

Comment:
if we hare allocating and adding drivers to this list that is separated from the one in drivers.json. Are they intended to be separate lists? Would that mean that one cannot consult platform drivers from ctrl? Maybe also that drivers are going to be loaded in state but not in platform? Maybe I'm missing something